tools/packaging/kernel/configs/fragments/arm64/serial.conf

```diff
@@ -1,3 +1,7 @@
 # This option is used for all 8250 compatible serial ports
 # that are probed through device tree.
 CONFIG_SERIAL_OF_PLATFORM=y
+
+# Enable Arm AMBA UART
+CONFIG_SERIAL_AMBA_PL011=y
+CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
```

Comment:
Please, also bump the kernel version with this change.

Reply:
Done, thanks.